.github/workflows/pr-patch-check-event.yml

```diff
@@ -31,18 +38,20 @@ jobs:
         with:
           github-token: ${{ steps.generate_token.outputs.token }}
           script: |
+            const {HEAD_REF, BASE_REF, REPO, SENDER} = process.env;
```

Comment:
You never import `SHA` and `PR_COMMIT_SHA` here.

Reply:
Thanks, fixed in 253e8af449e30b357c03e6ce62f8e9fcda5b3d80